scylla-server/src/services/systems.services.ts

```diff
@@ -1,11 +1,29 @@
 import prisma from '../prisma/prisma-client';
-import { ResponseFunction } from '../utils/message-maps.utils';
 
 /**
  * CRUD operation to get all systems with ResponseFunction type
- * @returns string contianing all the systems in the db
+ * @returns Promise<string> contianing all the systems in the db
  */
-export const getAllSystems: ResponseFunction = async () => {
+export async function getAllSystems(): Promise<string> {
   const data = await prisma.system.findMany();
   return JSON.stringify(data);
-};
+}
+
+/**
+ * CRUD opertation that creates system if it doesn't exist, otherwise does nothing.
+ * Currently designated private so not hooked up to server.
+ * @param system_name name of the system as string
+ * @returns Promise<void>
+ */
+export async function upsertSystems(system_name: string): Promise<void> {
```

Comment:
We dont really use function, i prefer to declare const upsertSystems = (system_name) => {}